public/css/presentation-style.css

```diff
@@ -1,33 +1,29 @@
-@import url("https://cdn.jsdelivr.net/npm/water.css@2/out/water.css");
-
-div.container{
+.container {
     display: grid;
     grid-template-rows: auto;
-	grid-template-columns: 60% 40%;
-	gap: 10%;
+    grid-template-columns: 60% 40%;
+    gap: 10%;
     align-items: center;
     overflow: hidden;
     background: #0096bfab;
     border-radius: 10px;
-    margin: 5% 7%; //outside box
```

Comment:
There comments are invalid css syntax, that was preventing all lines within the block from applying. Removed these comments and the following lines as they are not necessary